ExtractorUtils/Unstable/Runtime/Bootstrap.cs

```diff
@@ -0,0 +1,186 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Cognite.Extractor.Common;
+using Cognite.Extractor.Utils.Unstable.Tasks;
+using CogniteSdk;
+using CogniteSdk.Alpha;
+using Microsoft.Extensions.Logging;
+
+namespace Cognite.Extractor.Utils.Unstable.Runtime
```

Comment:
This file contains two separate classes, should they possibly be in two separate files?